Desktop_Interface/isodriver.cpp

```diff
@@ -255,125 +248,113 @@ void isoDriver::setVisible_CH2(bool visible){
     axes->graph(1)->setVisible(visible);
 }
 
-void isoDriver::setVoltageRange(QWheelEvent *event){
+void isoDriver::setVoltageRange(QWheelEvent* event)
+{
     if(doNotTouchGraph && !fileModeEnabled) return;
 
+    bool isProperlyPaused = properlyPaused();
```

Comment:
WIth a single pause state, the `properlyPaused` method can be eliminated and just m_paused can be used there.
Keep it as is for the moment, though.